### DISCUSSION TITLE
[Question]: TwelveLabs Embeddings

### DISCUSSION THREAD

**Discussion by nolandubeau** [danny-avila/LibreChat#2916] [Q&A]
### What is your question?

Hi,  I'm interested in using the the TwelveLabs embeddings to enable RAG over video files.  I was wondering how this might be achievable with the platform.  I see in the docs it references using HuggingFace.  Is it possible to have different vector db's through configuration?  Would it involve setting up something through LangChain? Thanks!

### More Details

Twelve Labs: https://www.twelvelabs.io/
Text Embeddings: https://docs.twelvelabs.io/reference/text-embeddings
Video Embeddings: https://docs.twelvelabs.io/reference/video-embeddings

### What is the main subject of your question?

Endpoints, Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by manish-tl** [comment]
@nolandubeau I am Manish, a PM at Twelve Labs. Thanks for trying out the new Embed API and providing valuable feedback. How are you using or plan to use the Video Embeddings currently? If you can share your usecase and workflow, that would be very helpful.
